src/index.html

```diff
@@ -12,13 +12,13 @@
 <body>
     <h1>Leaderboard</h1>
     <main class="content">
-    <article>
+    <article class="board">
    <div class="align">
         <h2 class="table-title">Recent scores</h2>
         <button type="button" class="refresh-button">Refresh</button>
     </div>
     <div class="table-section"></div>
```

Comment:
- [Optional]👍 Good job on displaying all the scores! However, there's a small improvement that can make the display even better. It's essential to show the top score at the top of the list to make it more user-friendly. To achieve this, I recommend sorting the scores in descending order before rendering them. By sorting the scores, the highest score will always be displayed at the top of the list, providing a better user experience and making it easier for users to find the top score. Keep up the great work, and happy coding! 🚀💻

Reply:
Yes, am definitely gonna work on this. Thanks for the highlight.